go/teams/list.go

```diff
@@ -61,9 +61,14 @@ func List(ctx context.Context, g *libkb.GlobalContext, arg keybase1.TeamListArg)
 
 	teamNames := make(map[string]bool)
 	upakLoader := g.GetUPAKLoader()
-	annotatedTeams := make([]keybase1.AnnotatedMemberInfo, len(teams))
+	annotatedTeams := []keybase1.AnnotatedMemberInfo{}
```

Comment:
how about `var annotatedTeams []keybase1.AnnotatedMemberInfo` so it's a proper `nil` when empty